newIDE/app/src/ResourcesList/LocalJfxrBridge.js

```diff
@@ -27,27 +27,29 @@ export const openJsfx = ({
     initialResourcePath.lastIndexOf('?cache=')
   );
 
-  const jsfxData = {
+  const jfxrData = {
     resourcePath: initialResourcePath,
     metadata: extraOptions.initialResourceMetadata,
     projectPath,
   };
 
-  ipcRenderer.removeAllListeners('jsfx-changes-saved');
-  ipcRenderer.on('jsfx-changes-saved', (event, newFilePath, fileMetadata) => {
+  ipcRenderer.removeAllListeners('jfxr-changes-saved');
+  ipcRenderer.on('jfxr-changes-saved', (event, newFilePath, fileMetadata) => {
     const resourcesManager = project.getResourcesManager();
     const resourceName = path.relative(projectPath, newFilePath); //Still needed for onChangesSaved()
+    console.log(resourceName)
```

Comment:
console.log to remove ;)

Reply:
ah sorry, I left it there because I still havent figured out why this:
https://github.com/4ian/GDevelop/pull/716#issuecomment-432407654
is hapening